koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -1029,11 +1029,22 @@ def get_source_build_info(self, build_id, build_nvr):
                 )
             raise koji.BuildError(err_msg)
 
+        build_type = self.session.getBuildType(koji_build['build_id'], strict=True)
```

Comment:
Can't we infer the type from `koji_build`?

Reply:
yeah